static/js/settings_bots.js

```diff
@@ -115,12 +117,18 @@ exports.set_up = function () {
             var bot_type = $('#create_bot_type :selected').val();
             var full_name = $('#create_bot_name').val();
             var short_name = $('#create_bot_short_name').val() || $('#create_bot_short_name').text();
+            var service_base_url = $('#create_service_base_url').val();
             var formData = new FormData();
 
             formData.append('csrfmiddlewaretoken', csrf_token);
             formData.append('bot_type', bot_type);
             formData.append('full_name', full_name);
             formData.append('short_name', short_name);
+
+            // If the selected bot_type is Outgoing webhook
+            if (bot_type === '3') {
```

Comment:
Would it make sense to define a constant `var OUTGOING_WEBHOOK_BOT_TYPE = 3` instead of using the hardcoded value 3?